plugins/summarize/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Changelog
 
 This changelog documents all notable changes to the summarize plugin for VAST.
+
+## 2.1.0
+
+### Features
+
+- The new `gather` action creates a list of all unique values in the selected
```

Comment:
What guided the choice for the name `gather`? I think other systems use `union` for this, no?

Reply:
I'm open to changing the name. This was the name used in the discussion that nerd-sniped me yesterday evening on whether this was possible, so I just went ahead and did it.

Reply:
I just renamed this to union.

Reply:
"union" is an operation defined on sets. When you explain what it does with ".. creates a list of all unique values.." you should really name it "unique" or "uniq". I prefer the former, but if our competitors agree on the latter we should use that.